### ISSUE TITLE
NTR: Pancreatic biopsy: acinar structures reduced (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9210]
New term request Pancreatic biopsy: acinar structures reduced (elthat_101205180316):   UMLS: 0267964

**Comment by pnrobinson** [comment]
@MickeySegal
I am not sure I understand what "acinar structures reduced" means? Can this be explained e.g. in the context of this review? [Pancreatic acinar cell: its role in acute pancreatitis](https://pubmed.ncbi.nlm.nih.gov/16423553/)

**Comment by MickeySegal** [comment]
We have this from https://www.ncbi.nlm.nih.gov/books/NBK1756/ "Exocrine pancreatic dysfunction resulting from severe depletion of pancreatic acinar cells is a classic feature of SDS" and https://ojrd.biomedcentral.com/articles/10.1186/1750-1172-5-29 "The pancreas is typically hypoplastic with a reduction of acinar tissue that exhibits evidence of necrosis".

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000157
- label: Decreased pancreatic acinar cell density
- parents: [Abnormal pancreas morphology (HP:0012090)](https://hpo.jax.org/app/browse/term/HP:0012090)
- pmid: PMID:32388592
- def: Number (count) of pancreatic acinar cells reduced below normal. This feature can be ascertained by pancreas biopsy.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9210
- synonyms: Pancreatic biopsy: acinar structures reduced(NONE)
